gammapy/estimators/parameter.py

```diff
@@ -345,5 +377,107 @@ def run(self, datasets, parameter):
             if "scan" in self.selection_optional:
                 result.update(self.estimate_scan(datasets, parameter))
 
+            if "sensitivity" in self.selection_optional:
+                result.update(self.estimate_sensitivity(datasets, parameter))
+
         result.update(self.estimate_counts(datasets))
         return result
+
+
+class ParameterSensitivityEstimator:
+    """Estimate the sensitivity to a given parameter
+
+    Computes the TS distribution in the non-null hypothesis using the
+    log likelihood of the Asimov dataset (i.e. a dataset with counts = npred)
+    and the non central chi2 distribution.
+    Once the TS distribution under the testing hypothesis is known,
+    one can compute the required parameter value
+    to have 50% of measurements above a given significance threshold.
```

Comment:
Shouldn't this be a parameter?
For instance the "brazilian flag" plots show the expectation (the 50% chance probability) along the 1 and 2 sigma ranges hence the 2.5%, 16%, 84%, and 97.5% chance proabilities

Reply:
I must say that I don't see where in the code this should appear. But I think at some point it should.

Reply:
The p-value is calculated in `ts_to_sigma` and setting ts=nc=ts_asimov with 1df gives 50%.
If we want the  (2.5%, 16%, 84%, and 97.5%) chance proabilities it should be equivalent run the evaluator with n_sigma - (-2,-1,+1+2)

Reply:
OK. 
I think it is fine for most of the n_sigma range but maybe not for all. Maybe we will have to check that.